pkg/lib/authn/identity/service/conversion.go

```diff
@@ -123,9 +123,10 @@ func anonymousFromIdentityInfo(i *identity.Info) *anonymous.Identity {
 
 func biometricToIdentityInfo(b *biometric.Identity) *identity.Info {
 	claims := map[string]interface{}{
-		identity.IdentityClaimBiometricKeyID:      b.KeyID,
-		identity.IdentityClaimBiometricKey:        string(b.Key),
-		identity.IdentityClaimBiometricDeviceInfo: b.DeviceInfo,
+		identity.IdentityClaimBiometricKeyID:               b.KeyID,
+		identity.IdentityClaimBiometricKey:                 string(b.Key),
+		identity.IdentityClaimBiometricDeviceInfo:          b.DeviceInfo,
+		identity.IdentityClaimBiometricFormattedDeviceInfo: b.FormattedDeviceInfo,
```

Comment:
My original thought was to call `deviceinfo.Format(b.DeviceInfo)` here. So we do not need to add a new field to `biometric.Identity`. On the other hand, we do not need to do additional things in biometricFromIdentityInfo because the new field does not exist.

Reply:
Oh, I got your point. Let me update it again.

Reply:
Updated!